templates/base/packages/nextjs/next.config.js.template.mjs

```diff
@@ -0,0 +1,33 @@
+import { withDefaults } from '../../../utils.js'
+
+const contents = ({ ignoreTsAndLintBuildErrors, extraConfig }) =>
+`// @ts-check
+
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  ${extraConfig[0]
+    ? `${Object.entries(extraConfig[0])
+        .map(([key, value]) => `${key}: ${JSON.stringify(value)},`)
+        .join('\n')}`
+    : ''}
```

Comment:
Let's update this with our new `stringify` util :)

Reply:
also to update: https://github.com/scaffold-eth/create-eth/pull/131#discussion_r1803458055

Reply:
Ohh yes lets update it here, I think `logoTitle` +  `logoSubTitle` makes more sense?

Reply:
We can also use 
```
Object.assign(
${stringify(extraconfig[0])}, 
{ reactStrictMode: true,
  typescript: {
    ignoreBuildErrors: ignoreTsAndLintBuildErrors,
  },
  eslint: {
    ignoreDuringBuilds: ignoreTsAndLintBuildErrors,
  },
  webpack: config => {
    config.resolve.fallback = { fs: false, net: false, tls: false };
    config.externals.push("pino-pretty", "lokijs", "encoding");
    return config;
  }}
)
```
but not sure it's better than `...`